.github/skills/create-jira-issue-from-user-input/scripts/env_utils.py

```
@@ -0,0 +1,34 @@
+    project-root ``.env`` is used: this skill's scripts are always invoked from
+    the workspace root, so ``Path.cwd() / ".env"`` is the project ``.env``. This
+    is intentional — no ancestor walk — so a missing project ``.env`` fails
+    loudly at Step 0 instead of silently loading an unrelated parent's file.
+    Existing process-env values take precedence (never overwritten).
+    """
+    candidate = Path(env_file) if env_file else Path.cwd() / ".env"
+
+    if candidate.is_file():
+        with candidate.open(encoding="utf-8") as fh:
+            for line in fh:
+                line = line.strip()
```

Ruff (0.15.20)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)